### PR TITLE
Features/5 star wars logo

### PR DESCRIPTION
**Pull request by szarlata**

### CODE REVIEW

**Review by DaveTheWebDev — COMMENTED** [review]
You can name variable "logo" not "p" and use const instead of let 🙂

File: index.html [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+        rel="stylesheet">
 </head>
-<body>
-
-<div id="swquiz-app">
-</div>
 
-<script src="src/index.js"></script>
+<body>
+    <div id="swquiz-app">
```
Comment (pspaczek) [inline index.html:17]: Recommendation: if you are using swquiz-app for js rendered content then you should remove img from it and move before 
```suggestion
    <img class="logo" src="static/assets/ui/StarWarsLogo.png">
    <div id="swquiz-app"></div>
```

File: src/index.js [hunk 2]
```diff
@@ -5,4 +5,14 @@ const ONE_SECOND_MILLIS = 1000;
...
+
+
+// Logo - reload page
+let p = document.querySelector('.logo'); 
+const refreshPage = () => {
+  location.reload();
+};
+p.addEventListener("click", refreshPage);
+
+
```
Comment (rpavlenko) [inline src/index.js:18]: I'm not sure that we need separate event listener for page reload. We can use usual `<a href="/">` tag with logo img inside instead.

Comment (szarlata) [reply src/index.js:18]: I'm not sure too, but it's js exercise, so I tried to do this in js way :P

**Pull request merged**